src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -35,37 +45,122 @@ export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
     }
 
     return html`
+      ${this.renderStyle()}
       <paper-input
         label="Title"
         value="${this._config!.title}"
         .configValue=${"title"}
+        .type=${"input"}
+        @value-changed="${this._valueChanged}"
+      ></paper-input>
+      <hui-theme-select-editor
+        .hass=${this.hass}
+        .value=${this._config!.theme}
+        .type=${"input"}
+        .configValue=${"theme"}
+        @change=${this._valueChanged}
+      ></hui-theme-select-editor><br>
+      <paper-input
+        label="Columns"
+        value="${this._config!.columns || ""}"
+        .configValue=${"columns"}
+        .type=${"input"}
         @value-changed="${this._valueChanged}"
-      ></paper-input><br>
+      ></paper-input>
+      <h4>Entities</h4>
+      <div class="entities">
+        ${this._configEntities!.map((entityConf, index) => {
+          return this.renderEntity(entityConf, index!);
+        })}
+      </div><br>
+      <paper-button
+        noink
+        raised
+        @click=${this.addEntity}
+      >Add Entity</paper-button>
+      <br><br>
       <paper-checkbox
         ?checked="${this._config!.show_name !== false}"
         .configValue=${"show_name"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's Name?</paper-checkbox><br><br>
       <paper-checkbox
         ?checked="${this._config!.show_state !== false}"
         .configValue=${"show_state"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's State Text?</paper-checkbox><br>
     `;
   }
 
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        .entities {
+          padding-left: 20px;
+        }
+      </style>
+    `;
+  }
+
+  private renderEntity(
+    entityConf: EntityConfig,
+    index: number
+  ): TemplateResult {
+    return html`
+      <ha-entity-picker
+        .hass="${this.hass}"
+        .value="${entityConf.entity || entityConf}"
+        .type=${"entity"}
+        .index="${index}"
+        @change="${this._valueChanged}"
+      ></ha-entity-picker>
+    `;
+  }
+
+  private addEntity() {
+    const newConfig = this._config!;
+    const newConfigEntites = this._configEntities!;
+
+    newConfigEntites.push(newConfigEntites[0].entity);
+    newConfig.entities = newConfigEntites;
+
+    fireEvent(this, "config-changed", {
+      config: newConfig,
+    });
+  }
+
   private _valueChanged(ev: MouseEvent): void {
     if (!this._config || !this.hass) {
       return;
     }
 
     const target = ev.target! as any;
+    let newConfig = this._config;
+    const newConfigEntites = this._configEntities!;
```

Comment:
you should make a copy since you're mutating this, and this value is part of the config we previously emitted.

```suggestion
    const newConfigEntites = this._configEntities!.concat();
```

Reply:
Does this make sure `newConfigEntities` is not an instance of `this._configEntities`?

Reply:
it makes a copy of the array.